evalai/challenges.py

```diff
@@ -264,18 +283,136 @@ def submit(ctx, file, annotation, large, public, private):
                 )
                 submission_metadata["project_url"] = click.prompt(
                     style("Project URL", fg="yellow"), type=str, default=""
                 )
                 submission_metadata["publication_url"] = click.prompt(
                     style("Publication URL", fg="yellow"), type=str, default=""
                 )
+            submission_meta_attributes = get_submission_meta_attributes(
+                ctx.challenge_id, ctx.phase_id
+            )
+            submission_attribute_metadata = []
+            if (
+                submission_meta_attributes
+                and len(submission_meta_attributes) > 0
+            ):
+                if click.confirm(
+                    "Do you want to include the Submission Metadata?"
+                ):
+                    for attribute in submission_meta_attributes:
+                        attribute_type = attribute["type"]
+                        attribute_name = attribute["name"]
+                        attribute_description = attribute["description"]
+                        attribute_required = attribute.get("required")
+                        if attribute_required:
+                            attribute_name = attribute_name + '*'
+                        value = None
+                        message = "{} ({})".format(
+                            attribute_name, attribute_description
+                        )
+                        if attribute_type == "text":
+                            while True:
+                                value = click.prompt(
+                                    style(message, fg="yellow"),
+                                    type=str,
+                                    default="",
+                                )
+                                if not attribute_required or value != "":
+                                    break
+                                echo(
+                                    "Error: {} is a required field".format(
+                                        attribute_name)
+                                )
+                        if attribute_type == "boolean":
+                            while True:
+                                value = click.prompt(
+                                    style(message, fg="yellow"), type=bool, default=""
+                                )
+                                if not attribute_required or value != "":
+                                    break
+                                echo(
+                                    "Error: {} is a required field".format(
+                                        attribute_name)
+                                )
+                        if attribute_type == "radio":
+                            while True:
+                                value = click.prompt(
+                                    style(
+                                        "{}: Choices:{}".format(
```

Comment:
@burnerlee can you add the `Choices:[]` to newline. Currently, if there are too many options it doesn't looks good